src/TLuaInterpreter.cpp

```diff
@@ -10381,7 +10381,11 @@ int TLuaInterpreter::sendIrc( lua_State * L )
 
     QPair<bool, QString> rval = mudlet::self()->mpIrcClientMap.value(pHost)->sendMsg(target, msg);
 
-    lua_pushboolean(L, rval.first);
+    if (rval.first) {
+        lua_pushboolean(L, true);
+    } else {
+        lua_pushnil(L);
```

Comment:
Is there anything in the Wiki about this - ah yes:
># sendIrc
>`sendIrc(target, message)`
    Sends a message to an IRC channel or person. Returns _true+status_ if message could be sent or was successfully processed by the client, or _nil+error_ if the client is not ready for sending, and _false+status_ if the client filtered the message or failed to send it for some reason. If the IRC client hasn't started yet, this function will initiate the IRC client and begin a connection.
>
>💡 **Note:** Since Mudlet 3.3, auto-opens the IRC window and returns the success status.
>### Parameters
> + `target`:
> nick or channel name and if omitted will default to the first available channel in the list of joined channels.
> + `message`:
> The message to send, may contain IRC client commands which start with / and can use all commands which are available through the client window.

So - I am not sure we can change this - else @vadi2 will be all over it and warning about maintaining backward compatibility with old scripts... 😜